coalib/parsing/DefaultArgParser.py

```diff
@@ -76,200 +76,122 @@ def default_arg_parser(formatter_class=None):
-                                   '--filter-by-language',
-                                   nargs='+',
-                                   metavar='LANG',
-                                   help="Filters the output of `--show-bears` "
-                                        "by the given languages.")
-
-        outputs_group.add_argument('-D',
-                                   '--show-description',
-                                   const=True,
-                                   action='store_const',
-                                   help="Shows bear descriptions for the "
-                                        "`--show-bears` output.")
-        outputs_group.add_argument('--show-details',
-                                   const=True,
-                                   action='store_const',
-                                   help='Shows bear details for the '
-                                        '`--show-bears` output, e.g '
-                                        'parameters and supported languages.')
+        outputs_group.add_argument(
+            '-B', '--show-bears', const=True, action='store_const',
+            help='list all bears')
+
+        outputs_group.add_argument(
+            '-l', '--filter-by-language', nargs='+', metavar='LANG',
+            help="filters `--show-bears` by the given languages")
+
+        outputs_group.add_argument(
+            '-D', '--show-description', const=True, action='store_const',
+            help="show bear descriptions for `--show-bears`")
+
+        outputs_group.add_argument(
+            '--show-details', const=True, action='store_const',
+            help='show bear details for `--show-bears`')
 
     # The following are "coala-json" specific arguments
     if parser_type == 'coala-json':
-        TEXT_LOGS_HELP = ("Don't display logs as json, display them as we "
-                          "normally do in the console.")
-        outputs_group.add_argument('--text-logs',
-                                   nargs='?',
-                                   const=True,
-                                   metavar='BOOL',
-                                   help=TEXT_LOGS_HELP)
-
-        OUTPUT_HELP = ('Write the logs as json to a file where filename is '
-                       'specified as argument.')
-        outputs_group.add_argument('-o',
-                                   '--output',
-                                   nargs=1,
-                                   metavar='FILE',
-                                   help=OUTPUT_HELP)
-
-        outputs_group.add_argument('-r',
-                                   '--relpath',
-                                   nargs='?',
-                                   const=True,
-                                   help="Return relative paths for files")
+        outputs_group.add_argument(
+            '--text-logs', nargs='?', const=True, metavar='BOOL',
+            help="use regular log messages instead of JSON")
```

Comment:
we should interchange the order of `--text-logs` and `--output` since we're referring "... instead of JSON" here IMO